src/test/java/teammates/test/cases/ui/ActionResultTest.java

```diff
@@ -16,5 +17,21 @@ public void testAppendParameters() {
         svr.addResponseParam(Const.ParamsNames.USER_ID, "david");
         assertEquals("/page/instructorHome?user=david", svr.getDestinationWithParams());
     }
+    
+    @Test
+    public void fileDownloadResult() {
```

Comment:
I believe we still use `test*` prefix so let's use it here (and the other test method) as well.